jep/12/README.adoc

```diff
@@ -36,24 +36,62 @@ endif::[]
 |===
 
 == Abstract
-This JEP provides an update to the existing Jenkins travel grant policy.
+
+The Jenkins project provides travel grants to a limited number of community members requesting assistance.
 
 == Specification
 
-[WARNING]
-====
-*(bitwiseman)*
-Rather than this being an update to Travel Grant Policy on the wiki,
-I'd like to see this replace the wiki page entirely.
-====
+The Jenkins project provides travel grants to a limited number of community members requesting assistance.
+Each travel grant supports one individual's travel to a Jenkins community event such as Jenkins User Conferences with up to 500 USD.
```

Comment:
I understand money is not unlimited but a I have attended the last two Jenkins user conferences and they cost upwards of around $3000 for the event in San Francisco for just flight and hotel.  This grant seems low for that as an example.

Reply:
@samrocketman 
I see your point.  At the same time, I don't think the project's budget has ever been sufficient to do more than what is outlined.  

It doesn't sound like your object to the intent of this PR, but want further changes to the program.

Can I ask you to raise this feedback on the discussion thread for this JEP or via a separate PR?

Reply:
I think that's addressed by https://github.com/jenkinsci/jep/pull/206/files#diff-e56f06fbe470a002ac6879a018b56756R93

Perhaps some internal inconsistency between this part and the linked?

Reply:
I think the change @daniel-beck mentions is good.  I mentioned the SF cost mainly because those who would use this kind of budget are likely in need of it.  That is to say they don’t have an alternate sponsor (company) and likely can’t afford to attend otherwise (personal finances, etc).  This was my primary concern with raising the point.